frappe/public/js/frappe/ui/toolbar/subscription.js

```diff
@@ -0,0 +1,80 @@
+$(document).on("startup", async () => {
+	if (!frappe.boot.setup_complete || !frappe.user.has_role("System Manager")) {
+		return;
+	}
+
+	let response = await frappe.xcall("frappe.utils.subscription.show_banner");
```

Comment:
Lets avoid this network call and put that info in boot directly. 


check `boot.py`